### PR TITLE
Add function to re-name USS file

### PR DESCRIPTION
**Pull request by kristinochka**

### CODE REVIEW

**Comment by ChrisBoehmCA** [comment]
Hi, you forgot to sign off your commit. Please fix your commit by issuing these commands: 


`git commit --amend --signoff`
`git push --force-with-lease origin add-uss-rename-func`

File: packages/zosfiles/src/api/methods/utilities/Utilities.ts [hunk 1]
```diff
@@ -111,4 +111,23 @@ export class Utilities {
...
+     * @param {string}          newFilePath - contains the new filepath
+     *
+     * @returns {Promise<Buffer>} Promise that resolves to json information
+     *
+     * @throws {ImperativeError}
+     */
+    public static async renameUSSFile(session: AbstractSession, USSFilePath: string, newFilePath: string): Promise<Buffer> {
+        ImperativeExpect.toNotBeNullOrUndefined(newFilePath, ZosFilesMessages.missingUSSFileName.message);
+        const oldFilePath = USSFilePath.charAt(0) === "/" ? USSFilePath : "/" + USSFilePath;
+        const payload = { request: "move",  from: path.posix.normalize(oldFilePath) };
```
Comment (kristinochka) [inline packages/zosfiles/src/api/methods/utilities/Utilities.ts]: `ZosFilesUtils.sanitizeUssPathForRestCall` removes the first forward slash in the path and URI encodes the path. Both of these things break the functionality.

**Comment by kristinochka** [comment]
@ChrisBoehmCA I made the changes you requested. I do not have access to Jenkins to see why the pipeline is failing.

File: packages/zosfiles/src/api/methods/utilities/Utilities.ts (hunk 1 above)
Comment (ChrisBoehmCA) [reply packages/zosfiles/src/api/methods/utilities/Utilities.ts]: OK. Makes sense

**Comment by ChrisBoehmCA** [comment]
Thanks for adding the system test. Passed for me

**Pull request merged**